api/google-manager/services/google-manager.js

```diff
@@ -56,6 +96,106 @@ class GoogleManager {
     }
   }
 
+  async createCalendar(title) {
+    try {
+      const calendar = await google.calendar({
+        version: 'v3',
+        auth: this.calendarAuth
+      });
+
+      const createdCalendar = await calendar.calendars.insert({
+        requestBody: {
+          summary: title
+        }
+      });
+      return createdCalendar.data;
+    } catch(err) {
+      console.error(`Google Calendar API returned error ${err} when creating calendar`);
+    }
+  }
+
+  async grantAcl(calendarId, email, role) {
+    try {
+      const calendar = await google.calendar({
+        version: 'v3',
+        auth: this.calendarAuth
+      });
+
+      await calendar.acl.insert({
+        calendarId,
+        requestBody: {
+          role: role,
+          scope: {
+            type: role === 'reader' ? 'group' : 'user',
```

Comment:
Why are we not using the role passed by the caller?

Reply:
Are you referring to the scope type? If so, I don't use the role the pass in because for the grantAcl function I pass in either 'reader' or 'owner' acl role. Depending on the acl role I make the scope type group or user since we're making it that google group has reader acl.

Reply:
Oh I see it expects user or group. I think we should make it explicit and ask the caller to define the type.

Reply:
> Oh I see it expects user or group. I think we should make it explicit and ask the caller to define the type.

so make another parameter that defines the scope type?

Reply:
Yes.